src/Razor/src/Microsoft.AspNetCore.Razor.Language/Components/ComponentBindLoweringPass.cs

```diff
@@ -735,32 +681,56 @@ private IntermediateNode[] RewriteUsage(IntermediateNode parent, BindEntry bindE
             // For general DOM attributes, we need to be able to create a delegate that accepts UIEventArgs
             // so we use 'CreateBinder'
             //
-            // EventCallbackFactory.CreateBinder(this, __value => <code> = __value, <code>) OR
-            // EventCallbackFactory.CreateBinder(this, __value => <code> = __value, <code>, <format>)
+            // EventCallbackFactory.CreateBinder(this, __value => <code> = __value, <code>, format: <format>, culture: <culture>)
             //
             // Note that the linemappings here are applied to the value attribute, not the change attribute.
-
-            string changeExpressionContent;
-            if (changeAttribute == null && format == null)
-            {
-                // DOM
-                changeExpressionContent = $"{ComponentsApi.EventCallback.FactoryAccessor}.{ComponentsApi.EventCallbackFactory.CreateBinderMethod}(this, __value => {original.Content} = __value, {original.Content})";
-            }
-            else if (changeAttribute == null && format != null)
+            if (changeAttribute == null)
             {
-                // DOM + format
-                changeExpressionContent = $"{ComponentsApi.EventCallback.FactoryAccessor}.{ComponentsApi.EventCallbackFactory.CreateBinderMethod}(this, __value => {original.Content} = __value, {original.Content}, {format.Content})";
+                changeExpressionTokens.Add(new IntermediateToken()
+                {
+                    Content = $"{ComponentsApi.EventCallback.FactoryAccessor}.{ComponentsApi.EventCallbackFactory.CreateBinderMethod}(this, __value => {original.Content} = __value, ",
```

Comment:
What is the advantage of splitting this in to multiple tokens? I don't see any line pragmas around these tokens in output.

Reply:
Just for symmetry with how its done in the rest of the code. I don't feel strongly about it.